client/src/modules/shopping-list/list.scss

```diff
@@ -0,0 +1,31 @@
+.archived-message {
+  position: relative;
+  width: 50%;
+  margin: 50px auto;
+  text-align: center;
+
+  &__header {
+    font-weight: bold;
+    font-size: 20px;
+    color: $text-normal;
+  }
+
+  &__button {
+    display: block;
+    width: 150px;
```

Comment:
![zrzut ekranu 2019-02-22 o 10 11 23](https://user-images.githubusercontent.com/27632432/53231907-3f7c2c80-368a-11e9-9bf6-e6a06c51478d.png)
Always setup desired button width and height with padding properties.